meta-lmp-base/recipes-bsp/efitools/efitools/unlock.patch

```diff
@@ -0,0 +1,201 @@
+From 54f1229af14365c12eb675401edd7132fef2b669 Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Tue, 1 Oct 2024 09:14:24 +0200
+Subject: [PATCH] Unlock feature
+
```

Comment:
Missing upstream-status flag.

Reply:
I don't think there is a lot of activity in efitools (last update was from 6 years ago). But we should have an upstream status flag anyway: https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html#patch-upstream-status